### PR TITLE
Provide all selected URIs for command execution

### PR DESCRIPTION
**Pull request by msujew**
#### What it does

Closes https://github.com/eclipse-theia/theia/pull/10198

VSCode provides in addition to the URI of the last selected file in the explorer also an array of all selected URIs. This change aligns Theia to this behavior.

#### How to test

1. Install the [test extension](https://github.com/msujew/theia-test-10198/raw/main/theia-test-10198-0.0.1.vsix) ([source](https://github.com/msujew/theia-test-10198/blob/main/theia-test-10198-0.0.1.vsix))
2. Select multiple/one file in the file explorer
3. Use the `Show URIs` command in the context menu of the file explorer
4. It should show the same output as if installed in vscode, including all selected URIs

#### Review checklist

- [x] As an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Review by vince-fugnitto — APPROVED** [review]
I confirm that the changes work as expected with the provided test extension 👍

**Comment by colin-grant-work** [comment]
@msujew, I merged #11290 ahead of this and created a conflict, sorry! Now this should be done in the plugin menu command adapter, rather than the menu contribution handler.

**Comment by msujew** [comment]
@colin-grant-work Thanks for the hint, I was able to rebase easily 👍

File: packages/plugin-ext/src/main/browser/menus/plugin-menu-command-adapter.ts [hunk 1]
```diff
@@ -245,13 +245,19 @@ export class PluginMenuCommandAdapter implements MenuCommandAdapter {
         return treeArgs;
     }
 
-    protected getSelectedResource(): [CodeUri | TreeViewSelection | undefined] {
+    protected getSelectedResource(): (CodeUri | CodeUri[] | TreeViewSelection | undefined)[] {
```
Comment (colin-grant-work) [inline packages/plugin-ext/src/main/browser/menus/plugin-menu-command-adapter.ts:248]: May I suggest this more exact typing?

```typescript
    protected getSelectedResources(): [CodeUri | TreeViewSelection | undefined, CodeUri[] | undefined] {
        const selection = this.selectionService.selection;
        const firstMember = TreeWidgetSelection.is(selection) && selection.source instanceof TreeViewWidget && selection[0]
            ? selection.source.toTreeViewSelection(selection[0])
            : (UriSelection.getUri(selection) ?? this.resourceContextKey.get())?.['codeUri'];
        const secondMember = TreeWidgetSelection.is(selection)
            ? UriSelection.getUris(selection).map(uri => uri['codeUri'])
            : undefined;
        return [firstMember, secondMember];
    }
```

**Pull request closed without merging**